src/sentry/integrations/github/client.py

```diff
@@ -62,20 +65,37 @@ def request(self, method, path, headers=None, data=None, params=None):
         return self._request(method, path, headers=headers, data=data, params=params)
 
     def get_token(self):
-        if not self.token or self.expires_at < datetime.datetime.utcnow():
+        """
+        Get token retrieves the active access token from the integration model.
+        Should the token have expried, a new token will be generated and
+        automatically presisted into the integration.
+        """
+        token = self.integration.metadata.get('access_token')
+        expires_at = self.integration.metadata.get('expires_at')
```

Comment:
Just a note. I think else where we'd been changing `expires_at` ti `expires`. I doesn't affect anything here though.